src/main/java/com/synopsys/integration/detect/tool/signaturescanner/BlackDuckSignatureScannerOptions.java

```diff
@@ -26,6 +26,7 @@ public class BlackDuckSignatureScannerOptions {
     @Nullable //Just to note that if you want server defaults this should be null.
     private final ReducedPersistence reducedPersistence;
 
+    @Nullable
     private final boolean uploadSource;
```

Comment:
Type needs to change to Boolean class too
```suggestion
    private final Boolean uploadSource;
```

Reply:
Done ✅